docs/user/managing-devices.md

```diff
@@ -661,7 +661,19 @@ To connect, use the `flightctl console` command specifying the device's name, an
 flightctl console device/<some_device_name>
 ```
 
-To disconnect, enter "exit" on the console. To force-disconnect, press `<ctrl>+b` three times.
+To disconnect, enter "exit" on the console.
```

Comment:
Does this mean we no longer have a way to recover from a hanging console?

Reply:
Since we can send binary information through the console, we cannot assume a certain sequence causes disconnection.
SSH doesn't have such sequence.
To disconnect a session there are the following possibilities:
- Kill the console session locally
- Connect with another console session to the device and kill the session.

Reply:
> SSH doesn't have such sequence.
It does: The escape sequence is `Enter`, `~`, `.`.  See, e.g., [here](https://foosel.net/til/about-ssh-escape-sequences/).

Reply:
OK, so we need to distinguish between valid binary input and such escape sequence.
The previous approach [scans](https://github.com/flightctl/flightctl/blob/76486ae190a6d9eebfd54b93498ea43f8d298e39/internal/cli/console.go#L195-L202)  the input.  It is not practical for tool supporting uploads.

Reply:
There are a couple ways to fix this:
1. SSH only escapes if there's a newline before "~.", reducing the chance of accidentally killing the connection.  It also contains two characters, whereas ours contains one.
2. When running in non-interactive mode, disable this functionality.  SSH does this when executing remote commands as well (e.g., ssh host "cmd").

Reply:
Only the second is a valid option, since when uploading a file, we cannot assume any sequence is unexpected.
Will implement only for interactive mode.

Reply:
Will change to "~." since <ctrl>-B is used by "less" and "vim" as page-up.